hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/DecommissionStatusSubCommand.java

```diff
@@ -139,4 +165,48 @@ private void printCounts(DatanodeDetails datanode, JsonNode counts, int numDecom
       System.err.println("Error getting pipeline and container counts for " + datanode.getHostName());
     }
   }
+
+  private Map<String, Object> getDatanodeDetails(DatanodeDetails datanode) {
+    Map<String, Object> detailsMap = new LinkedHashMap<>();
+    detailsMap.put("uuid", datanode.getUuid().toString());
+    detailsMap.put("networkLocation", datanode.getNetworkLocation());
+    detailsMap.put("ipAddress", datanode.getIpAddress());
+    detailsMap.put("hostName", datanode.getHostName());
```

Comment:
```suggestion
    detailsMap.put("hostname", datanode.getHostName());
```
This will match with same as we have in `org.apache.hadoop.ozone.recon.api.types.DatanodeMetadata`